alchemist-implementationbase/src/main/kotlin/it/unibo/alchemist/model/implementations/terminators/StableForSteps.kt

```diff
@@ -47,6 +47,7 @@ class StableForSteps<T>(
 ) : Predicate<Environment<T, *>> {
     private var success: Long = 0
     private var positions: Map<Node<T>, Position<*>> = emptyMap()
+    @Transient
     private var contents = makeTable<T>(0)
```

Comment:
what happens on deserialization?

Reply:
I'm not quite sure, the problem was a ClassNotFoundException:
```
it.unibo.alchemist.model.implementations.terminators.StableForSteps$Companion$$Lambda$1058/0x0000000800885440
```
what I supposed was that being the class Initialized locally and then distributed, the remote node can't find the "instance" of the lambda function in the `makeTable` function.

Reply:
I mean: that "contents" goes to null upon deserialization. It is at a risk of throwing NPE

Reply:
"contents" is only used in equality check, and eventually get overwritten. 
The first time it may compare with null instead of an empty table and then with the table with the actual contents. If a user wants to terminate after 1 stable step, it won't be accurate.